src/software/ai/hl/stp/tactic/move/move_fsm.h

```diff
@@ -27,6 +27,8 @@ struct MoveFSM
         AutoChipOrKick auto_chip_or_kick;
         // The maximum allowed speed mode
         MaxAllowedSpeedMode max_allowed_speed_mode;
+        // The target spin while moving in revolutions per second
```

Comment:
nit; I'd recommend using radians per second, as it's more standard unit, and certainly what I'd assume `rps` meant. 

I'd also strongly recommend spelling out the units a bit more explicitly in the name (ex. `target_spin_rev_per_s` or `target_spin_rad_per_s`), especially because `rps` could easily be either.

Reply:
Similar comment goes for other instances of `rps` in this PR.

Reply:
changed to rev_per_s